knife-vsphere.gemspec

```diff
@@ -14,7 +14,7 @@ Gem::Specification.new do |s|
   s.rubygems_version = "1.6.2"
   s.homepage = 'http://github.com/ezrapagel/knife-vsphere'
   s.license = 'Apache'
-  s.add_dependency('netaddr', ["~> 1.5.0"])
-  s.add_dependency('chef', [">= 0.10.0"])
-  s.add_dependency('rbvmomi', ["~> 1.5.1"])
+  s.add_dependency('netaddr', ['~> 1.5', '>= 1.5.0'])
+  s.add_dependency('chef', ['~> 12.0', '>= 12.0.3'])
+  s.add_dependency('rbvmomi', ['~> 1.8', '>= 1.8.0'])
```

Comment:
The rbvmomi thing has bugged me for a while, thanks for fixing that.

Is the update to Chef required for this PR to work?